packages/app/app/actions/scrobbling.js

```diff
@@ -127,3 +136,38 @@ export function updateNowPlayingAction(artist, track, session) {
       });
   };
 }
+
+export function FmFavImport(){
+  let importStatus = store.get('lastFm.lastFmFavImportStatus');
+  if (importStatus){
+    store.set('lastFm.lastFmFavImportStatus', false);
+    return dispatch => {
+      dispatch({
+        type: LASTFM_DISABLE_IMPORT,
+        payload: null
+      });
+    };
+  } else {
+    store.set('lastFm.lastFmFavImportStatus', true);
+    return dispatch => {
+      dispatch({
+        type: LASTFM_ENABLE_IMPORT,
+        payload: null
+      });
+    };
+  }
+
+}
+
+export function FmFavUpdateMsg(msg) {
```

Comment:
We don't need to store this message persistently, or in redux, since it can be generated in the component.